src/archivist/services/documentTypes.json

```diff
@@ -1,191 +1,198 @@
+      "audience": "shareholders",
+      "object": "human rights"
+    }
+  },
+  "In-App Purchases Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "in-app and other virtual goods purchase usage"
+    }
+  },
+  "Review Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "review process of provided content"
+    }
+  },
+  "Brand Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "developer",
+      "object": "usage of the service provider’s brand"
+    }
+  },
+  "Quality Guidelines": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "quality of content produced by means of the service"
+    }
+  },
+  "Data Controller Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "end user personal data"
+    }
+  },
+  "Data Processor Agreement": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "status of data processor (in the sense of GDPR) for the service provider"
+    }
+  },
+  "User Consent Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "data controllers (in the sense of GDPR)",
+      "object": "user consent collection and storage"
+    }
+  },
+  "Closed Captioning Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "closed captioning on streamed content"
+    }
+  },
+  "Seller Warranty": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "protection of end users that act as sellers"
+    }
+  },
+  "Single Sign-On Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "connecting user account to other services"
+    }
+  },
+  "Vulnerability Disclosure Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "vulnerability reporter",
+      "object": "how to report a security vulnerability or issue"
+    }
+  },
+  "Live Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "end user’s live service usage"
+    }
+  },
+  "Ad Publishing Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
+      "object": "acceptable and unacceptable usage"
+    }
+  },
+  "Complaints Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "anyone",
+      "object": "how generic complaints will be handled"
+    }
+  },
+  "Transport Policy": {
+    "commitment": {
+      "writer": "service provider",
+      "audience": "end user",
```

Comment:
The ICAO indeed [uses the term “user”](https://www.icao.int/sustainability/Documents/ConsumerInterests_B.pdf) in their definition, but also uses “passenger”. The CIT in the [International Convention for the transportation of Passengers](https://cit-rail.org/secure-media/files/documentation_de/passenger/civ/civ1999-f-d-e.pdf?cid=224515) uses “passenger”. I believe it is more precise and more specific to use this term 🙂

```suggestion
      "audience": "passenger",
```